backend/db/utils.py

```diff
@@ -24,6 +24,34 @@
 SCHEMA = CONFIG["schema"]
 
 
+# TODO: automatically determine latest backup name rather than param
+# TODO: format this to return something better. Right now looks like:
+#  [('code_sets', 'test2_n3c', 1), ('code_sets', 'test_n3c', 1), ('concept_set_container', 'test2_n3c', 1), ('concept_set_container', 'test_n3c', 1), ('concept_set_members', 'test2_n3c', 1), ('concept_set_members', 'test_n3c', 1), ('concept_set_version_item', 'test2_n3c', 1), ('concept_set_version_item', 'test_n3c', 1)]
+# todo: pass params (not working right now for some reason) to run_sql() instead of string interpolating, e.g.: WHERE schemaname in (:schema_names)...
+def qc_check_db_counts(
```

Comment:
@Sigfried FYI: QC check that compares table rows, where I left it off. Needs a little more work (check the todo's above the func)